examples/bri-3/.env.sample

```diff
@@ -4,10 +4,12 @@
 # Prisma supports the native connection string format for PostgreSQL, MySQL, SQLite, SQL Server, MongoDB and CockroachDB.
 # See the documentation for all the connection string options: https://pris.ly/d/connection-strings
 
+# Explanation of the params can be found in the Readme file.
+
 DATABASE_URL="postgresql://postgres:example@localhost:5432/postgres"
 GOERLI_RPC_URL=""
-GOERLI_SERVICE_DID="did:ethr:0x5:0x0aE6F58120E3e53E94FeB7BFeAA11C7CFF6F70F0"
-GEORLI_SERVICE_SIGNER_PRIVATE_KEY=""
+GOERLI_SERVICE_DID="did:ethr:0x5:<bpi_subject_public_key>"
+GOERLI_SERVICE_SIGNER_PRIVATE_KEY="<bpi_subject_private_key>"
```

Comment:
this is not private key of bpi subject, it is private key of service (bpi) that is signing transactions

so dids are being resolved from specified smart contract on goerli net, i assume this private key is if something needs to be added to those dids, for now we are just resolving

will try to document this more precisely, but it is not bpi subject private key

Reply:
Ok, this is more clear now, updated the readme a bit